examples/clinical_pipeline/clinical_processing_pipeline.py

```diff
@@ -1,48 +1,153 @@
 
-from fortex.elastic import ElasticSearchPackIndexProcessor
-from fortex.huggingface.bio_ner_predictor import BioBERTNERPredictor
-from fortex.huggingface.transformers_processor import BERTTokenizer
+from forte.processors.writers import PackIdJsonPackWriter
+from ft.onto.base_ontology import Sentence, EntityMention
+
 from fortex.nltk import NLTKSentenceSegmenter
 
-def main(input_path: str, output_path: str, max_packs: int = -1):
-    pl = Pipeline[DataPack]()
-    pl.set_reader(
-        Mimic3DischargeNoteReader(), config={"max_num_notes": max_packs}
-    )
-    pl.add(NLTKSentenceSegmenter())
+from ftx.medical.clinical_ontology import (
+    NegationContext,
+    MedicalEntityMention,
+    MedicalArticle,
+)
+from fortex.health.processors.negation_context_analyzer import (
+    NegationContextAnalyzer,
+)
+from fortex.health.processors.icd_coding_processor import ICDCodingProcessor
+
+
+def get_json(path: str):
+    file_obj = open(path)
+    data = json.load(file_obj)
+    file_obj.close()
+    return data
+
+
+def update_stave_db(
+    default_project_json, chat_project_json, chat_doc_json, config
+):
+    project_id_base = 0
+    with StaveSession(url=config.Stave.url) as session:
+        session.login(username=config.Stave.username, password=config.Stave.pw)
+
+        projects = session.get_project_list().json()
+        project_names = [project["name"] for project in projects]
+
+        if (
+            default_project_json["name"] in project_names
+            and chat_project_json["name"] in project_names
+        ):
+
+            base_project = [
+                proj
+                for proj in projects
+                if proj["name"] == default_project_json["name"]
+            ][0]
+            return base_project["id"]
 
-    config = yaml.safe_load(open("bio_ner_config.yml", "r"))
+        resp1 = session.create_project(default_project_json)
+        project_id_base = json.loads(resp1.text)["id"]
+
+        resp2 = session.create_project(chat_project_json)
+        project_id_chat = json.loads(resp2.text)["id"]
+
+        chat_doc_json["project_id"] = project_id_chat
+        doc_id = session.create_document(chat_doc_json)
+        project_list = session.get_project_list().json()
+
+    return project_id_base
+
+
+def main(
+    input_path: str, output_path: str, max_packs: int = -1, run_ner_pipeline=0
+):
+    print("Starting demo pipeline example..")
+    config = yaml.safe_load(open("clinical_config.yml", "r"))
     config = Config(config, default_hparams=None)
 
-    pl.add(BERTTokenizer(), config=config.BERTTokenizer)
-    pl.add(BioBERTNERPredictor(), config=config.BioBERTNERPredictor)
-    pl.add(ElasticSearchPackIndexProcessor())
-
-    pl.add(
-        PackIdJsonPackWriter(),
-        {
-            "output_dir": output_path,
-            "indent": 2,
-            "overwrite": True,
-            "drop_record": True,
-            "zip_pack": True,
-        },
+    if run_ner_pipeline == 1:
```

Comment:
Why are we placing two scripts into this one `main` function? One is this NER pipeline, and the other is the search pipeline. Just make them two scripts so that we don't have to worry about to use this `run_ner_pipeline` parameter